datafusion/sql/src/unparser/rewrite.rs

```diff
@@ -300,3 +301,27 @@ fn find_projection(logical_plan: &LogicalPlan) -> Option<&Projection> {
         _ => None,
     }
 }
+
+pub struct TableAliasRewriter {
```

Comment:
How about adding some doc for this public struct?

Reply:
Sure